Update trello.md with Capabilities information (#2939)
n8n-io/n8n-docs: docs/integrations/builtin/credentials/trello.md

9. Select **Generate a new API Key**.
10. Copy the **API key** from Trello and enter it in your n8n credential.
11. In your Trello API key page, enter your n8n base URL as an **Allowed origin**.

> 12. In Capabilities make sure to select the necessary options.

Reviewer: ```suggestion
12. In **Capabilities** make sure to select the necessary options.
```

13. Select the **Token** link next to your Trello **API Key**.

> 14. When prompted, select **Allow** to grant all th epermissions it asks for.

Reviewer: ```suggestion
14. When prompted, select **Allow** to grant all the permissions asked for.
```
